src/onnx/parse_resize.cpp

```diff
@@ -138,8 +135,8 @@ calc_neighbor_points(const std::vector<std::vector<std::vector<std::size_t>>>& v
                            return dim;
                        });
     }
-
-    return calc_neighbor_points(vvv_ind, i_dim + 1, vec_dims1, in_s);
+    vec_dims = std::move(vec_dims1);
+    return calc_neighbor_points(vvv_ind, i_dim + 1, vec_dims, in_s);
```

Comment:
I think it would be simpler to do how you showed yesterday:

```cpp
vec_dims.clear();
return calc_neighbor_points(vvv_ind, i_dim + 1, std::move(vec_dims1), in_s);
```

This way we wont need double moves.